lib/ace/editor.js

```diff
@@ -766,7 +785,7 @@ var Editor = function(renderer, session) {
         }
     };
 
-    this.onSelectionChange = function(e) {
+    this.onSelectionChange = function() {
```

Comment:
... and this change is probably harmless, but still why? Code cleanup and removal of unused variable is not the goal now.